tests/kafkatest/services/performance/streams_performance.py

```diff
@@ -31,6 +31,54 @@ def __init__(self, test_context, kafka, numrecs, load_phase, test_name, num_thre
                                                             test_name,
                                                             num_threads)
 
+        self.load_phase = load_phase
+
+        if self.load_phase == "false":
+            JmxMixin.__init__(self, num_nodes=1,
```

Comment:
nit: should `num_nodes=1` be it's own line?

Reply:
ack.